src/main/webapp/i18n/de/guidedTourCourseExerciseOverview.json

```diff
@@ -57,32 +52,18 @@
             },
             "repositoryCreated": {
                 "headline": "Dein Repository wurde erstellt",
-                "content": "Dein Repository ist bereit und kann ab jetzt geklont werden. Fahre mit dem nächsten Schritt fort, um mehr zu erfahren."
+                "content": "Dein Repository ist bereit und kann ab jetzt geklont werden. Fahre mit dem nächsten Schritt fort, um mehr über das Klonen und Bearbeiten von Programmieraufgaben auf Artemis zu erfahren."
             },
             "cloneRepository": {
-                "headline": "Klone das Repository in SourceTree",
-                "content": "Schau dir das folgende Video an, um zu verstehen, wie man ein Git Repository in SourceTree klont. Danach solltest du auf das <tt>X</tt> oben rechts klicken, um das Tutorial zu stoppen und die im Video angezeigten Schritte durchzuführen. Danach kannst du das Tutorial wieder von deinem Konto-Menü aus fortsetzen.",
-                "videoUrl": "https://www.youtube.com/embed/0YYxgs14S6A?rel=0"
-            },
-            "inspectSourceTree": {
-                "headline": "Lerne SourceTree kennen",
-                "content": "Schau dir das folgende Video, um die grundlegenden Funktionen von SourceTree kennenzulernen.",
-                "videoUrl": "https://www.youtube.com/embed/Piu_UBK6d7c?rel=0"
-            },
-            "importEclipse": {
-                "headline": "Importiere das Java Projekt in Eclipse",
-                "content": "Schau dir das folgende Video an, um zu sehen, wie man ein Projekt in Eclipse importiert.",
-                "videoUrl": "https://www.youtube.com/embed/5VrNNlEuCBo?rel=0"
-            },
-            "inspectProject": {
-                "headline": "Untersuche das Projekt",
-                "content": "In diesem Video schauen wir uns das Projekt an und lösen die darin gegebene Aufgabe.",
-                "videoUrl": "https://www.youtube.com/embed/K-mX03gmNSk?rel=0"
-            },
-            "commitAndPush": {
-                "headline": "Reiche deine Änderungen ein",
-                "content": "Schau dir das folgende Video an, um zu verstehen, wie man Änderungen committed und pushed.",
-                "videoUrl": "https://www.youtube.com/embed/sjT-S6AhQvM?rel=0"
+                "headline": "Klone das Repository in IntelliJ",
+                "content": "Schau dir das folgende Video an, um zu verstehen, wie man ein Git Repository in IntelliJ klont. Danach solltest du auf das <tt>X</tt> oben rechts klicken, um das Tutorial zu stoppen und die im Video angezeigten Schritte durchzuführen. Danach kannst du das Tutorial wieder von deinem Benutzerkonto-Menü (Login-Name oben rechts) aus fortsetzen.",
+                "hint": "Falls du noch nicht die Java JDK 17 installiert hast, wird auch im Video gezeigt, wie du diese erhältst.",
```

Comment:
For me, it's das JDK but I don't know if there's an objectively correct solution.